node-operators/networks/collators/activities.md

```diff
@@ -146,54 +147,54 @@ If you want to temporarily leave the candidate pool, you can easily do so using
 
 ## Change Self-Bond Amount {: #change-self-bond-amount }
 
-As a candidate, changing your self-bond amount varies slightly depending on if you're bonding more or less. If you're bonding more, it is a straightforward process where you can increase your stake via the `candidateBondMore()` extrinsic. You do not have to wait any delays and you do not need to schedule a request and then execute it, instead your request will be executed instantly and automatically.
+As a candidate, changing your self-bond amount varies slightly depending on whether you're bonding more or less. If you're bonding more, it is a straightforward process where you can increase your stake via the `candidateBondMore()` extrinsic. You do not have to wait for any delays, and you do not need to schedule a request and then execute it; instead, your request will be executed instantly and automatically.
 
-If you wish to bond less, you have to schedule a request, wait an [exit delay](#collator-timings), and then you will be able to execute the request and get a specified amount of tokens back into your free balance. In other words, scheduling the request doesn't decrease the bond instantly or automatically, it will only be decreased once the request has been executed.
+If you wish to bond less, you have to schedule a request, wait for an [exit delay](#collator-timings), and then you will be able to execute the request and get a specified amount of tokens back into your free balance. In other words, scheduling the request doesn't decrease the bond instantly or automatically; it will only decrease once the request has been executed.
 
 ### Bond More {: #bond-more }
 
-As a candidate, there are two options for increasing one's stake. The first and recommended option is to send the funds to be staked to another owned address and [delegate to your collator](/tokens/staking/stake/#how-to-nominate-a-collator). Alternatively, collators that already have at least the [minimum self-bond amount](/node-operators/networks/collators/requirements/#minimum-collator-bond){target=_blank} staked can increase their bond from [Polkadot.js Apps](https://polkadot.js.org/apps/?rpc=wss://wss.api.moonriver.moonbeam.network#/accounts). Navigate to the **Developer** tab, click on **Extrinsics**, and follow these steps:
+As a candidate, there are two options for increasing one's stake. The first and recommended option is to send the funds to be staked to another owned address and [delegate them to your collator](/tokens/staking/stake/#how-to-nominate-a-collator). Alternatively, collators that already have at least the [minimum self-bond amount](/node-operators/networks/collators/requirements/#minimum-collator-bond){target=_blank} staked can increase their bond through [Polkadot.js Apps](https://polkadot.js.org/apps/?rpc=wss://wss.api.moonbase.moonbeam.network#/accounts). Navigate to the **Developer** tab, click on **Extrinsics**, and follow these steps:
```

Comment:
We should include a warning here that increasing your collator bond, while possible, does not increase rewards for collators. The only way to increase their rewards is to stake funds from another address. I'm not aware of any situations in which a collator would want to increase their bond without getting additional rewards.

Reply:
Wouldn't one reason be if the collator was trying to make it into the active set and needed to increase their total bond amount?

Reply:
@themacexpert @eshaben I believe that collator's bond account as self-stake. Meaning that if they increase their bond, it is accounted as them increasing the stake in theirself

https://github.com/moonbeam-foundation/moonbeam/blob/master/pallets/parachain-staking/src/lib.rs#L1937-L1952

Reply:
Yeah, that's what I thought. That it would increase their self-bond, therefore their total bond. I wasn't 100% sure what impact increasing their self-bond amount had on their rewards